index.html

```diff
@@ -41,6 +41,122 @@ <h4>Lets connect</h4>
+                  <ul>
+                   <li>html</li>
+                   <li>css</li>
+                   <li>javascript</li>
+                 </ul>
+                 <button class="see">see project</button>
+                </div>
+
+              </article>
+
+              <article class="project">
+                <div class="project-box">
+                 <img src="./images/Snapshoot3.png" alt="">
+                 <h3>Tonic</h3>
+                 <h4><span>CANOPY</span>&nbsp;
+                   <img src="./images/Counter.png" alt="">&nbsp;
+                   Back End Dev&nbsp;
+                   <img src="./images/Counter.png" alt="">
+                   &nbsp; 2015
+                 </h4>
+                 <p>A daily selection of privately<br> personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                 <ul>
+                   <li>html</li>
+                   <li>css</li>
+                   <li>javascript</li>
+                 </ul>
+                 <button class="see">see project</button>
+                </div>
+
+              </article>
+
+              <article class="project">
+                <div class="project-box">
+                 <img src="./images/Snapshoot4.png" alt="">
+                 <h3>Multi-Post<br>Stories</h3>
+                 <h4><span>CANOPY</span>&nbsp;
+                   <img src="./images/Counter.png" alt="">&nbsp;
+                   Back End Dev&nbsp;
+                   <img src="./images/Counter.png" alt="">
+                   &nbsp; 2015
+                 </h4>
+                 <p>A daily selection of privately<br> personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                 <ul>
+                   <li>html</li>
+                   <li>css</li>
+                   <li>javascript</li>
+                 </ul>
+                 <button class="see">see project</button>
+                </div>
+              </article>
+            </section>
+
+              <section class="about">
+                <div class="about-container">
+                  <h2>About <br>myself</h2>
+                  <p>Hello I’m a software developer! I can help you build a product , feature
+                    or website Look through some of my work and experience! If you like what
+                    you see and have a project you need coded, don’t hestiate to contact me.</p>
+                  <p class="connect">LET'S CONNECT</p>
+                  <ul class="socials about-socials" >
+                    <li><a href="#"><img src="./images/Twitter.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/Linkedin.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/medium.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/github.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/angelist.svg" alt=""></a></li>
+                  </ul>
```

Comment:
Your icons here are not align to the center. I would kindly advise you use `font awesome` just like you have done in `<ul class="socials">`
|Figma Design|Your Design|
|---|---|
|![image](https://user-images.githubusercontent.com/12239821/170738442-0af95bb9-e078-46a5-9445-daa0cb58c4c2.png)|![image](https://user-images.githubusercontent.com/12239821/170738596-f6247341-c429-4cdb-9342-570add5838b9.png)|
|![image](https://user-images.githubusercontent.com/12239821/170738497-4c8cfddc-1182-4c4d-ab78-70c2bf426dc4.png)|![image](https://user-images.githubusercontent.com/12239821/170738841-22e6b66c-8ecc-4941-a711-9081be2bec19.png)|

|